contracts/pl.json

```diff
@@ -0,0 +1,37 @@
+{
+  "generic_goodluck": "Powodzenia!",
+  "generic_objectives": "Aby zawrzeć tę umowę, musisz:",
+  "generic_requirements": "Ten kontrakt pojawi się w grze, jeśli spełnisz następujące wymagania:",
+  "generic_reward": "Za ukończenie tego kontraktu otrzymasz:",
```

Comment:
The french/english translations give me a weird sentence, so to be sure. Does it mean that's what you get when you end the contract ?

Reply:
Yes. It means "For resolving this contract you will recieve:"